app/routes.ts

```diff
@@ -21,6 +22,7 @@ const typeOfAppealController = setupTypeOfAppealController();
 const devNextPageController = setupDevNextPageController();
 const personalDetailsController = setupPersonalDetailsController();
 const contactDetailsController = setupContactDetailsController();
+const setConfirmationRouter = setConfirmationController();
```

Comment:
Do you mean setConfirmationController? just so we follow the same convention everywhere...